spec/commands/newgem_spec.rb

```diff
@@ -144,6 +144,23 @@ def gem_skeleton_assertions(gem_name)
         expect(bundled_app("test_gem/README.md").read).not_to include("github.com/bundleuser")
       end
     end
+
+    context "git is not installed in the system" do
+      before do
+        `rm "#{ENV["GIT_CONFIG"]}"` if File.exist?(ENV["GIT_CONFIG"])
+        reset!
+        in_app_root
+        bundle "gem #{gem_name}"
+        remove_push_guard(gem_name)
+      end
+
+      it "contribute URL set to [USERNAME]" do
+        expect(bundled_app("test_gem/README.md").read).to include("[USERNAME]")
+        expect(bundled_app("test_gem/README.md").read).not_to include("github.com/bundleuser")
+      end
+
+      it_should_behave_like "git config is absent"
```

Comment:
@colby-swandale this line is using [shared example](https://github.com/bundler/bundler/pull/5466/files/4dec0e6f2c980c2b21fc96c987fbda5c6c351093#diff-3fbe0dfd8b4d505d057982532848c10eR66) to check the `gemspec` file. It must put `"TODO: Write your name"` , `"TODO: Write your email` in the place of author name, email (same as when git config is unset).